analyses/snv-callers/scripts/03-calculate_tmb.R

```diff
@@ -220,19 +190,32 @@ if (opt$tcga) {
   # Format two fields of metadata for use with functions
   metadata <- readr::read_tsv(opt$metadata) %>%
```

Comment:
Ah. You can't see it in looking at just the changes here,  but would be able to see it if you open up the full file (Please let me know if its not clear there either). There's an `opt$tcga` which funnels an if statement so that whatever is specified as the `opt$metadata` gets handled properly. Aka, if opt$tcga == TRUE, then this is done, but if it is false, then the metadata is processed how we do for PBTA. there's no way to really automate the metadata processing since the metadata files for pbta and tcga are so different.

Reply:
Ah okay, I understand. 
The comment at the beginning of the R script:
> --metadata : Relative file path to MAF file to be analyzed.

made me think that the metadata file here may have been processed in a previous script and that is the file you are using here, since you specified it to be a MAF file. 

Now that you have clarified this a bit, I restructure the description of the `--metadata` file at the top of the script in order to emphasize that the metadata file is unique to the tcga/pbta MAF file.

Reply:
I think that comment you quoted had been wrong for quite sometime and we didn't notice! It should say `Relative file path to metadata file to be analyzed` Classic copy and paste mistake on my part.